src/collectives/device/common.h

```diff
@@ -41,13 +41,13 @@ static __device__ void load_parallel(void* dst, void* src, size_t size, int tid)
   int* s = (int*)src;
   for (int o = tid; o < (size/sizeof(int)); o += blockDim.x) d[o] = s[o];
 }
-static __device__ void load_coll(struct ncclWork* localWork, struct ncclWork* hostWork, int tid, struct ncclDevComm* comm) {
+static __device__ void load_coll(struct ncclWork* localWork, struct ncclWork* hostWork, int tid, struct ncclDevComm* comm, int rbid) {
```

Comment:
Do non-sckl callers break with the extra argumnet?  If so, consider making this a default param of -1 and dealing with it in the code.

Reply:
No this is fine, scklNumBlocksPerChannel is set to 1 for non-sckl algorithms.

Reply:
This will be fixed in the next version.